.github/workflows/build-release-binary.yml

```diff
@@ -206,22 +206,34 @@ jobs:
 
       - name: Build maker docker image
         run: |
+          VERSION=${{ github.event.release.tag_name }}
+
+          if [ "$VERSION" = "preview" ]; then
+              VERSION=latest
+          fi
```

Comment:
Something like this might be cleaner: https://stackoverflow.com/a/3953666/2489334